zoneminder/zoneminder-event-cleanup.sh

```diff
@@ -30,20 +38,28 @@ for id in $(echo ${archived[*]}); do
 done
 
 # Find events according to our filter
-find ${zmcache}/ -mindepth 2 -type l -mtime +${olderthan} ${findexception[*]} -exec ls -la {} + > $tmpfile
+find ${zmcache}/ -mindepth 3 -maxdepth 3 -type d -mtime +${olderthan} ${findexception[*]} -exec ls -ld {} + > $tmpfile
```

Comment:
Looks good! Just double-checked with my Zoneminder 1.34 version and this should work. However another parameter could help users of "old installations" to continue use the script. Example: user uses the parameter `-o` (for old/outdated) and the script will use the old `find` command with the symlinks.

Reply:
Hmm..., that's a good point.
i will update the script to use optional a variable to declare, it's for Zoneminder with Version 1.20 and lower.

Reply:
i have added in the user settings *zm_linked_version*.
So we can now change the script to cleanup the old version von Zoneminder or the new one (default is blank).